1.19.0

matteobruni opened the pull request:
refactor: renamed backgroundMode to fullScreen

matteobruni committed:
feat: upgraded components
refactor: renamed backgroundMode to fullScreen

ghost commented:
## DeepCode's analysis on [#60cdba](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/b6c06ee17d4a0bba9bb1266a2c1a0ed60a0fafc5/matteobruni/tsparticles/60cdba758620a392d24f8ae253a1c8cf6c59c476/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=2&) found:

- :information_source: **2** minor issues. :point_down:
- :heavy_check_mark: **2** issues were fixed.
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">'public' is implicit. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/60cdba758620a392d24f8ae253a1c8cf6c59c476/core/main/src/Options/Classes/Options.ts#L24">Options.ts:24</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/60cdba758620a392d24f8ae253a1c8cf6c59c476/core/main/src/Options/Classes/Options.ts#L33">Options.ts:33</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/60cdba758620a392d24f8ae253a1c8cf6c59c476/core/main/src/Options/Classes/Options.ts#L40">Options.ts:40</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/b6c06ee17d4a0bba9bb1266a2c1a0ed60a0fafc5/matteobruni/tsparticles/60cdba758620a392d24f8ae253a1c8cf6c59c476/pr/_/%2Fcore%2Fmain%2Fsrc%2FOptions%2FClasses%2FOptions.ts/javascript%2Ftslint%2Fmember-access/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
    <tr>
      <td width="77%">No catch method for promise. This may result in an unhandled promise rejection. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/60cdba758620a392d24f8ae253a1c8cf6c59c476/components/preact/src/Particles.tsx#L107">Particles.tsx:107</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/60cdba758620a392d24f8ae253a1c8cf6c59c476/components/preact/src/Particles.tsx#L109">Particles.tsx:109</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/60cdba758620a392d24f8ae253a1c8cf6c59c476/components/vue3/src/Particles/Particles.vue#L63">Particles.vue:63</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/b6c06ee17d4a0bba9bb1266a2c1a0ed60a0fafc5/matteobruni/tsparticles/60cdba758620a392d24f8ae253a1c8cf6c59c476/pr/_/%2Fcomponents%2Fpreact%2Fsrc%2FParticles.tsx/javascript%2Fdc_interfile_project%2FPromiseNotCaughtGeneral/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/b6c06ee17d4a0bba9bb1266a2c1a0ed60a0fafc5/matteobruni/tsparticles/60cdba758620a392d24f8ae253a1c8cf6c59c476/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=2&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

code-factor committed:
[CodeFactor] Apply fixes to commit 47060c2

[ci skip] [skip ci]

matteobruni committed:
build: updated lerna configuration (like v2)

matteobruni committed:
build: updated angular component README and demo

matteobruni committed:
build: updated three dependency for security issue

matteobruni committed:
build: fixed ci build

matteobruni committed:
chore(release): published new version

 - ng-particles@2.2.0
 - inferno-particles@1.2.0
 - jquery-particles@1.19.0
 - preact-particles@1.19.0
 - react-tsparticles@1.19.0
 - svelte-particles@1.2.0
 - particles.vue@2.2.0
 - particles.vue3@1.2.0
 - tsparticles-editor@1.19.0
 - tsparticles@1.19.0
 - ng-particles-demo@1.4.0
 - editor-demo@1.4.0
 - inferno-particles-demo@1.2.0
 - jquery-particles-demo@1.4.0
 - tsparticles-demo@1.4.0
 - preact-particles-demo@1.4.0
 - react-tsparticles-demo@1.4.0
 - svelte-demo@1.2.0
 - particles.vue-demo@1.4.0
 - particles.vue3-demo@1.2.0
 - tsparticles-preset-60fps@1.4.0
 - tsparticles-preset-background-mask@1.4.0
 - tsparticles-preset-basic@1.4.0
 - tsparticles-preset-big-circles@1.4.0
 - tsparticles-preset-bouncing@1.4.0
 - tsparticles-preset-fire@1.4.0
 - tsparticles-preset-font-awesome@1.4.0
 - tsparticles-preset-fountain@1.4.0
 - tsparticles-preset-snow@1.4.0
 - tsparticles-preset-stars@1.4.0
 - tsparticles-shape-bubble@1.4.0
 - tsparticles-shape-heart@1.4.0
 - tsparticles-shape-multiline-text@1.4.0
 - tsparticles-shape-spiral@1.4.0
 - cra-template-particles-typescript@1.2.0
 - cra-template-particles@1.2.0
 - tsparticles-website@1.19.0

matteobruni committed:
build: fixed READMEs and wiki urls

matteobruni committed:
build: fixed READMEs and wiki urls

matteobruni committed:
build: fixed READMEs and wiki urls

matteobruni committed:
build: website demo.js updated to 1.19.0

matteobruni committed:
build: website updated to 1.19

matteobruni committed:
build: website updated to 1.19

matteobruni committed:
build: fixed samples page, updated to 1.19.0

matteobruni committed:
build: updated templates to 1.19.0
